### DISCUSSION TITLE
Zed updates pyright on every launch?

### DISCUSSION THREAD

**Discussion by Solarunit** [zed-industries/zed#20510] [Feature Requests]
I noticed that Zed updates **pyright** on every launch? Why is that? Is there an option to disable auto updates?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support

**Reply by Solarunit** [reply]
well, I am not on Discord and do not plan to go there. Should we expect the closing of the whole GitHub Discussions?
